app/mailers/invisiblize_mailer.rb

```diff
@@ -0,0 +1,25 @@
+class InvisiblizeMailer < ApplicationMailer
+  def to_developer
```

Comment:
This will break the notification unfortunately. I originally hade `developer` but the noticed notification kept trying to call it's own `developer` method

Reply:
That's what it looks like 🤷‍♂️ 
This:
```ruby
deliver_by :email, mailer: "InvisiblizeMailer", method: :developer
```
causes a `NoMethodError` saying that `:to_sym` isn't a method on the `Developer` model instance, but using `:to_developer` works fine

Reply:
That's frustrating. Thanks for documenting – we might need to open an issue in Noticed.